src/components/spreadsheet-view/spreadsheet/spreadsheet-toolbar/global-model-editor/spreadsheet-model-global-editor-table.tsx

```diff
@@ -78,6 +69,20 @@ export function SpreadsheetModelGlobalEditorTable() {
                         },
                     }),
             },
+            {
+                label: intl.formatMessage({ id: 'spreadsheet/global-model-edition/column_id' }),
+                dataKey: COLUMN_ID,
+                type: DndColumnType.CUSTOM,
+                editable: true,
+                initialValue: '',
+                showErrorMsg: true,
+                width: '20%',
+                maxWidth: '20%',
+                component: (rowIndex) =>
+                    ColumnIDEditor({
+                        name: `${COLUMNS_MODEL}[${rowIndex}].${COLUMN_ID}`,
+                    }),
```

Comment:
```suggestion
                component: (rowIndex) =>
                    TableTextInput({
                        name: `${COLUMNS_MODEL}[${rowIndex}].${COLUMN_ID}`,
                        inputProps: {
                            autoFocus: true,
                        },
                    }),
```
You can do this without creating a new component

Reply:
:+1: 
Removed